Gateway/Request/OneclickAuthorizationDataBuilder.php

```diff
@@ -69,13 +69,9 @@ public function build(array $buildSubject)
 
         $payment->unsAdditionalInformation(AdyenOneclickDataAssignObserver::ENCRYPTED_SECURITY_CODE);
 
-        if ($payment->getAdditionalInformation('customer_interaction')) {
-            $shopperInteraction = "Ecommerce";
-        } else {
-            $shopperInteraction = "ContAuth";
-        }
-
-        $requestBody['shopperInteraction'] = $shopperInteraction;
+        // We override the previously set shopperInteraction in the buildRecurringData() to ContAuth here because if
+        // it's a later payment we should always send ContAuth
```

Comment:
Just for more clarity 
```suggestion
        // it's a payment with a stored method we should always send ContAuth
```